CHANGELOG.md

```diff
@@ -4,10 +4,14 @@ IMPROVEMENTS:
 
 * build: Update Nomad verison to 1.10.0 [GH-431](https://github.com/hashicorp/nomad-driver-podman/pull/431)
 * build: Updated to Go 1.24.2 [[GH-430](https://github.com/hashicorp/nomad-driver-podman/pull/430)]
+* config: Adds support for oom_score_adj [GH-425](https://github.com/hashicorp/nomad-driver-podman)
```

Comment:
It looks like we have brackets around all the others. (Except the one at the top, which we should fix while we're here.)

```suggestion
* config: Adds support for oom_score_adj [[GH-425](https://github.com/hashicorp/nomad-driver-podman)]
```